src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/AuthModal.java

```diff
@@ -26,36 +26,29 @@
   private WebElement forgottenPasswordLink;
 
   private WebDriver webDriver;
+  private String mainWindowHandle;
 
   public AuthModal(WikiaWebDriver webDriver){
     super();
     this.webDriver = webDriver;
-
+    waitForNewWindow();
+    this.mainWindowHandle =  this.webDriver.getWindowHandle();
```

Comment:
I would recommend to move the switching part to different method, constructor is not the best place to do it now